.prettierrc

```diff
@@ -1 +1 @@
-{ "printWidth": 200, "singleQuote": true, "tabWidth": 4, "semi": true, "trailingComma": "all" }
+{ "printWidth": 200, "singleQuote": true, "tabWidth": 4, "semi": true, "trailingComma": "all", "jsxSingleQuote": true }
```

Comment:
ngl made me feel guilty seeing single quotes in JSX

Reply:
will remove haha

Reply:
No no, please I'm just not used to it 😂 by all means we can be consistent with all single quotes

Reply:
Ok sounds good haha I'll leave it in